pages/services/cassandra/2.6.0-3.11.4/getting-started/index.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: layout.pug
+navigationTitle: Getting Started
+excerpt: Getting started with Cassandra
+title: Getting Started
+menuWeight: 20
+model: /services/cassandra/data.yml
+render: mustache
+---
+
+Getting started with a test instance of the DC/OS {{ model.techName }} service is straightforward.
+
+## Prerequisites
+
+- Depending on your security mode in Enterprise DC/OS, you may need to [provision a service account](/services/{{ model.serviceName }}/2.6.0-3.11.4/security/#provisioning-a-service-account) before installing. You will be able to create the service account only if you have a `superuser` permission.
+	- `strict` [security mode](/latest/security/ent/#security-modes) requires a service account.
+	- `permissive` security mode a service account is optional.
+	- `disabled` security mode does not require a service account.
+- Your cluster must have at least {{ model.install.minNodeCount }} private nodes.
+{{ model.install.customRequirements }}
+
+<p class="message--note"><strong>NOTE: </strong>The default DC/OS Apache {{ model.techShortName }} installation provides reasonable defaults for trying out the service, but may not be sufficient for production use. You may require different configurations depending on the context of the deployment.</p>
+
+
+## Installation from the DC/OS CLI
+
+To start a basic test cluster, run the following command on the DC/OS CLI. Enterprise DC/OS users must follow additional instructions. More information about installing DC/OS Apache {{ model.techShortName }} on Enterprise DC/OS is available on the [Authenticating DC/OS Services](/1.11/security/ent/service-auth/custom-service-auth/) documentation. 
+
+```shell
+dcos package install {{ model.serviceName }}
+```
+You can specify a custom configuration in an `options.json` file and pass it to `dcos package install` using the `--options` parameter.
+
+```
+$ dcos package install {{ model.serviceName }} --options=<options>.json
+```
+
+It is recommended that this custom configuration is stored in source control.
+
+For more information about building the `options.json` file, see the [DC/OS documentation](/1.11/deploying-services/config-universe-service/) for service configuration access.
+
+## Installation from the DC/OS Web Interface
+
+You can [install DC/OS Apache {{ model.techShortName }} from the DC/OS web interface](/1.11/deploying-services/install/). If you install DC/OS Apache {{ model.techShortName }} from the web interface, you must install the DC/OS Apache {{ model.techShortName }} CLI subcommands separately. From the DC/OS CLI, enter the following command:
```

Comment:
Would it be appropriate to update the link here from the 1.11/deploying-services pages to the 1.13 or 1.14 versions?